src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -82,6 +82,59 @@ std::string postUpdateContext
     return answer;
   }
 
+  //
+  // If upcr.contextElementResponseVector contains >= TWO responses and the second of them is a 'SccFound',
+  // then skip the first, the one that isn't 'SccFound'.
+  // Also, if the first of them is 472, then skip the second
+  //
+  // FIXME P6: Temporary hack for 'Entity found but Attribute not found':
+  //           See FIXME P6 'Temporary hack ...' 
+  //           in MongoCommonUpdate.cpp, function processContextElement
+  //
+
+
+  //
+  // Removing 'garbage' from the response vector (when just *one* incoming contextElement
+  //
+  
+  if (upcr.contextElementResponseVector.size() != parseDataP->upcr.res.contextElementVector.size())
+  {
+    for (unsigned int ix = 0; ix < upcr.contextElementResponseVector.size() - 1; ++ix)
+    {
+      //
+      // 2 cases:
+      //
+      // o 1. 472+302:  keep 302
+      //   Out[0]: (472) request parameter is invalid/not allowed
+      //   Out[1]: (302) Found
+      //
+      // o 2. 472+404: keep 472
```

Comment:
Actually, it is needed, as a 'biproduct' of the forwarding:

A non-found attribute now gets a 472, but docs it not counted up, so that we'll get into the 'if (docs == 0)' part.  In that part, if the attribute is found in a registration a 302 is added but if not, then a 404 is added.

So, this 404 must be removed and in the case of 'found', the 472 must be removed.

I can add what I just wrote here as a comment in the code ...